packages/node-resolve/src/index.js

```diff
@@ -17,7 +17,7 @@ import {
 const builtins = new Set(builtinList);
 const ES6_BROWSER_EMPTY = '\0node-resolve:empty.js';
 const nullFn = () => null;
-const defaults = {
+export const DEFAULT_OPTIONS = {
```

Comment:
Not down with this change. We might want to expose this as something else, but the name internally shouldn't change. Let's keep `defaults` here.

Reply:
This will lead to having `defaults` in the es build and having `DEFAULT_OPTIONS` in cjs build, no? 🤔

Reply:
The wrapper should be the source of truth for both. Sorry, I thought that was the way this was going. I can check out your fork tomorrow and take a look at making this happen tomorrow.

Reply:
That’s not what @NotWoods was saying.
He made me create a separate `cjs-wrapper`.
I can revert and create the wrapper back in `index.js` if you want

Reply:
@shellscape We should pick a single direction for this so @MichaelDeBoey doesn't have to run in circles :)

Using the wrapper for ES modules breaks the usual import format of `import default, { others } from 'module';`. The wrapper is only meant for CommonJS because it mutates the default export instead of exporting two separate items. We can find some alternative solution if preferred, such as making a breaking change and only exporting named exports like the Terser plugin. (`import { nodeResolve, DEFAULT_OPTIONS } from 'module'`)

Reply:
As suggested in #299, this was meant as a non-breaking change, but I can export both as named if you want

Reply:
Yeah I'm sorry for the pivot suggestion, but the wrapper started to feel wrong the more I looked at it. Let's do a new major, nothing wrong with that. And do two named exports. I really don't want to change the internal name of defaults, but I'm good to export it as something else - that much doesn't matter to me. I do think we should only export a deeply frozen copy of that though.

Reply:
So I export both `nodeResolve` & `defaults` (freezed) as named exports? 🤔